src/server/profiles/account/account.controller.js

```diff
@@ -0,0 +1,130 @@
+/* eslint no-unused-vars: ["error", { "argsIgnorePattern": "app" }] */
+const { resolveFromVersion } = require('../../utils/resolve.utils');
+const responseUtils = require('../../utils/response.utils');
+const errors = require('../../utils/error.utils');
+
+module.exports.getAccount = function getAccount ({ profile, logger, config, app }) {
+	let { serviceModule: service } = profile;
+
+	return (req, res, next) => {
+		let { version } = req.sanitized_args;
+		// Get a version specific resource
+		let Account = require(resolveFromVersion(version, 'uscore/Account'));
+
+		return service.getAccount(req.sanitized_args, logger)
+			.then((results) =>
+				responseUtils.handleBundleReadResponse( res, version, Account, results, {
+					resourceUrl: config.auth.resourceServer
+				})
+			)
+			.catch((err) => {
+				logger.error(err);
+				next(errors.internal(err.message, version));
+			});
+	};
+
+};
+
+
+module.exports.getAccountById = function getAccountById ({ profile, logger, app }) {
+	let { serviceModule: service } = profile;
+
+	return (req, res, next) => {
+		let { version } = req.sanitized_args;
+		// Get a version specific resource
+		let Account = require(resolveFromVersion(version, 'uscore/Account'));
+
+		return service.getAccountById(req.sanitized_args, logger)
+			.then((results) =>
+				responseUtils.handleSingleReadResponse(res, next, version, Account, results)
+			)
+			.catch((err) => {
+				logger.error(err);
+				next(errors.internal(err.message, version));
+			});
+	};
+};
+
+/**
+ * @description Controller for creating Account
+ */
+module.exports.createAccount = function createAccount ({ profile, logger, app }) {
+	let { serviceModule: service } = profile;
+
+	return (req, res, next) => {
+		let { version, resource_body, resource_id } = req.sanitized_args;
+		// Get a version specific resource
+		let Account = require(resolveFromVersion(version, 'uscore/Account'));
+		// Validate the resource type before creating it
+		if (Account.__resourceType !== resource_body.resourceType) {
+			return next(errors.invalidParameter(
+				`'resourceType' expected to have value of '${Account.__resourceType}', received '${resource_body.resourceType}'`,
+				version
+			));
+		}
+		// Create a new resource and pass it to the service
+		let newResource = new Account(resource_body);
```

Comment:
newResource should be snake case since it's a var and probably should be const

Reply:
looks like everything is using let so maybe just leave it unless we want to change all of the other ones too

Reply:
Ok. Will do snake case.